apps/web/app/(app)/onboarding/components/onboarding.tsx

```diff
@@ -0,0 +1,165 @@
+"use client";
+
+import { ConnectWithFormbricks } from "@/app/(app)/onboarding/components/inapp/ConnectWithFormbricks";
+import { InviteTeamMate } from "@/app/(app)/onboarding/components/inapp/InviteTeamMate";
+import Objective from "@/app/(app)/onboarding/components/inapp/SurveyObjective";
+import Role from "@/app/(app)/onboarding/components/inapp/SurveyRole";
+import { CreateFirstSurvey } from "@/app/(app)/onboarding/components/link/CreateFirstSurvey";
+import { Session } from "next-auth";
+import { useEffect, useState } from "react";
+
+import { TEnvironment } from "@formbricks/types/environment";
+import { TTeam } from "@formbricks/types/teams";
+import { TUser } from "@formbricks/types/user";
+
+import PathwaySelect from "./PathwaySelect";
+import { OnboardingHeader } from "./ProgressBar";
+
+interface OnboardingProps {
+  isFormbricksCloud: boolean;
+  session: Session;
+  environment: TEnvironment;
+  user: TUser;
+  team: TTeam;
+  webAppUrl: string;
+}
+
+export function Onboarding({
+  isFormbricksCloud,
+  session,
+  environment,
+  user,
+  team,
+  webAppUrl,
+}: OnboardingProps) {
+  const [selectedPathway, setSelectedPathway] = useState<string | null>(null);
+  const [progress, setProgress] = useState<number>(16);
+  const [formbricksResponseId, setFormbricksResponseId] = useState<string | undefined>();
+  const [CURRENT_STEP, SET_CURRENT_STEP] = useState<number | null>(null);
```

Comment:
Please follow our variable naming convention and name them `currentStep` & `setCurrentStep`